src/org/openstreetmap/josm/plugins/osmobjinfo/OSMObjInfoActions.java

```diff
@@ -1,26 +1,34 @@
 package org.openstreetmap.josm.plugins.osmobjinfo;
 
+import static org.openstreetmap.josm.tools.I18n.tr;
+
 import java.awt.Toolkit;
 import java.awt.datatransfer.Clipboard;
 import java.awt.datatransfer.StringSelection;
+import java.util.List;
+
 import javax.swing.JOptionPane;
+
+import org.openstreetmap.josm.data.Bounds;
+import org.openstreetmap.josm.gui.MainApplication;
 import org.openstreetmap.josm.gui.Notification;
-import static org.openstreetmap.josm.tools.I18n.tr;
+import org.openstreetmap.josm.plugins.osmobjinfo.OSMObjInfotDialog.AllOsmObjInfo;
 import org.openstreetmap.josm.tools.OpenBrowser;
 
 /**
  *
  * @author ruben
  */
 public class OSMObjInfoActions {
+    public static final String COPY = "Copy: {0}";
 
     public static void copyUser(String user) {
         if (!user.isEmpty()) {
             String linkUser = "https://www.openstreetmap.org/user/" + user;
             StringSelection selection = new StringSelection(linkUser);
             Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
             clipboard.setContents(selection, selection);
-            new Notification(tr("Copy: " + linkUser)).setIcon(JOptionPane.INFORMATION_MESSAGE).setDuration(Notification.TIME_SHORT).show();
+            new Notification(tr(COPY, linkUser)).setIcon(JOptionPane.INFORMATION_MESSAGE).setDuration(Notification.TIME_SHORT).show();
```

Comment:
Translations do not work this way, you have to mark strings for translations using `marktr()`

Reply:
I'll fix this.